devops/ci/multi_node_deploy/Jenkinsfile_3Node_Deploy

```diff
@@ -133,14 +133,25 @@ node {
                 sshCommand remote: remotes[0], command: """
                     yum install -y yum-utils
                     yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
-                    yum install --nogpgcheck -y python3 python36-m2crypto salt salt-master salt-minion
-                    rm -rf /etc/yum.repos.d/*3rd_party*.repo
                     yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
+                    
+                    cat <<EOF >/etc/pip.conf
+                    [global]
+                    timeout: 60
+                    index-url: $CORTX_RELEASE_REPO/python_deps/
+                    trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
```

Comment:
the same question here

Reply:
Yes @andkononykhin2, it's a new requirement that follows the cortx-preq rpm from RE.